index.html

```diff
@@ -11,7 +11,7 @@
 <body>
 	<nav class="nav-links"> 
```

Comment:
I can't see a closing tag for the <nav> (although it still appears to be functioning properly so have I missed something?)